db/version_set.cc

```diff
@@ -475,9 +475,10 @@ bool Version::OverlapInLevel(int level,
 
 int Version::PickLevelForMemTableOutput(
     const Slice& smallest_user_key,
-    const Slice& largest_user_key) {
+    const Slice& largest_user_key,
+    const int level_limit) {
   int level = 0;
-#if 0
+#if 1
```

Comment:
corrected as recommended

> On Aug 8, 2015, at 4:11 PM, Paul A. Place <notifications@github.com> wrote:
> 
> In db/version_set.cc <https://github.com/basho/leveldb/pull/163#discussion_r36581331>:
> 
> >    int level = 0;
> > -#if 0
> > +#if 1
> Since this code is being put back into production, should remove the "#if 1/#endif" conditional compilation statements.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/basho/leveldb/pull/163/files#r36581331>.
>